### PR TITLE
To Do list: interactive list

### PR DESCRIPTION
**Pull request by gergitokusse**
## The following are the feature added to the project(to-do list).

- New Javascript file that contains a method for status updates named Interactive was created.
- Add event listener to checkbox and clear complete task.
- All updated data storage to local storage

### CODE REVIEW

File: src/Interactiv.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+  static clearcomplt() {
+    let mylocaldata = JSON.parse(localStorage.getItem('toDoList'));
+    if (mylocaldata === null) { mylocaldata = []; }
+    const completearry = [];
+    mylocaldata.forEach((element, i) => {
+      if (mylocaldata[i].complete === true) {
+        completearry.push(mylocaldata[i]);
+        mylocaldata.splice(i, 1);
+      }
```
Comment (mukhammadrizooff) [inline src/Interactiv.js]: - [ ] [REQUIRED] Good job so far! Kindly when I try to tick the tasks, this part cannot be set properties of null. Please fix it 🙏🏼 

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @gergitokusse, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitHub Flow ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Interactiv.js (hunk 1 above)
Comment (gergitokusse) [reply src/Interactiv.js]: # 🙏🙏🙏 Thanks, I will fix it
## 🤝🤝🤝
### 👍👍👍

Comment (KDlamini) [reply src/Interactiv.js]: Well resolved ✔️

**Review by KDlamini — APPROVED** [review]
### Hi @gergitokusse 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Great PR title and description
✔️ Methods related to the status updates (`completed`: `true` / `false`) are implemented
✔️ The task object's values for `completed` is updated upon user actions
✔️ The "Clear all completed" function  method is well-implemented
✔️ Nice touch updating indexes after completed tasks are cleared 🔥 

# Status: Approved ✔️ 

Great-job-smiley

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**